Source/SwiftLintFramework/Rules/ForceCastRule.swift

```diff
@@ -26,8 +26,7 @@ public struct ForceCastRule: Rule {
         ruleName: "Force Cast Rule",
         ruleDescription: "This rule checks whether you don't do force casts.",
         nonTriggeringExamples: [
-            "NSNumber() as? Int\n",
```

Comment:
`verifyRule()` will treat all triggering examples as non-triggering when commented out. So `// NSNumber() as! Int` was already being tested because `NSNumber() as! Int` is a triggering example. So this line was duplicating a test.